README.md

```diff
@@ -6,6 +6,51 @@ Nevertheless, hopefully the guide can serve as training material to iOS app deve
 
 > __Just like any software, this guide will rot unless we update it. We encourage everyone to help us on that, just open an issue or send a pull request!__
 
+## Jaibroken Test
+
+Checking whether a device is jailbroken or not . Attacker can run tools like Cycript, GDB, Snoop-it etc to perform runtime analysis and steal sensitive data from within your application. This will help for extra layer of security for your application, you should not allow your application to be run on a jailbroken device.
+
+> **Audit tip:** Must ensure the application not working on jaibroken device. Use below code for jaibroken test.
+
+``` Objective c sample code snippet.
++(BOOL)isJailbroken{
+ 
+#if !(TARGET_IPHONE_SIMULATOR)
+ 
+if ([[NSFileManager defaultManager] fileExistsAtPath:@"/Applications/Cydia.app"]){
+return YES;
+}else if([[NSFileManager defaultManager] fileExistsAtPath:@"/Library/MobileSubstrate/MobileSubstrate.dylib"]){
+return YES;
+}else if([[NSFileManager defaultManager] fileExistsAtPath:@"/bin/bash"]){
+return YES;
+}else if([[NSFileManager defaultManager] fileExistsAtPath:@"/usr/sbin/sshd"]){
+return YES;
+}else if([[NSFileManager defaultManager] fileExistsAtPath:@"/etc/apt"]){
+return YES;
+}
+ 
+NSError *error;
+NSString *stringToBeWritten = @"This is a test.";
+[stringToBeWritten writeToFile:@"/private/jailbreak.txt" atomically:YES
+encoding:NSUTF8StringEncoding error:&amp;error];
+if(error==nil){
+//Device is jailbroken
+return YES;
+} else {
+[[NSFileManager defaultManager] removeItemAtPath:@"/private/jailbreak.txt" error:nil];
+}
+ 
+if([[UIApplication sharedApplication] canOpenURL:[NSURL URLWithString:@"cydia://package/com.example.package"]]){
+//Device is jailbroken
+return YES;
+}
+#endif
+ 
+//All checks have failed. Most probably, the device is not jailbroken
```

Comment:
Can you make sure that the code is correctly formatted? (space after // and indentation)

Please also expand in the comment that this is only an heuristic and it cannot be determine with certainty that the device is NOT jailbroken.